spec/unit/rust-crypto/rust-crypto.spec.ts

```diff
@@ -92,6 +92,16 @@ describe("RustCrypto", () => {
             const res = await rustCrypto.preprocessToDeviceMessages(inputs);
             expect(res).toEqual(inputs);
         });
+
+        it("should pass through one time key counts", async () => {
+            const oneTimeKeyCounts = new Map<string, number>([["signed_curve25519", 50]]);
+            await expect(rustCrypto.preprocessOneTimeKeyCounts(oneTimeKeyCounts)).resolves.not.toBeDefined();
+        });
+
+        it("should pass through unused fallback keys", async () => {
+            const unusedFallbackKeys = new Set(["signed_curve25519"]);
+            await expect(rustCrypto.preprocessUnusedFallbackKeys(unusedFallbackKeys)).resolves.not.toBeDefined();
+        });
```

Comment:
these tests don't seem to test what they claim to test!

Reply:
(it's probably not necessary to test that they *actually* pass through the key counts/fallback keys: we have that in the integration tests. But we shouldn't have tests that claim to test things that they do not.)